.github/dependabot.yml

```
@@ -8,7 +8,7 @@ updates:
     ignore:
       # Restrict updates to jetty in the 9.4.x space
```

Now that we are ignoring versions greater than or equal to 11 rather than greater than or equal to 10, this comment is now incorrect and needs to be updated:

```suggestion
      # Restrict updates to jetty in the 10.x space
```